view/js/jquery-textcomplete/jquery.textcomplete.min.js

```diff
@@ -1,3 +1,5 @@
+// @license magnet:?xt=urn:btih:d3d9a9a6595521f9666a5e94cc830dab83b65699&dn=expat.txt Expat
```

Comment:
What's the Expat license?

Reply:
It's the MIT license, but it seems that the MIT license is officially the Expat license and the GNU people want the Expat license listed. Wikipedia says

> A common form of the MIT License is this (taken from the Open Source Initiative's website; this is identical to the "Expat License", and different from the license used in the X source code):